src/components/admin/form-components/withSelector.jsx

```diff
@@ -151,18 +151,18 @@ export const withSelector = Finder => {
         name: PropTypes.string.isRequired,
       }),
     ).isRequired,
-    onChange: PropTypes.func.isRequired,
     onUpdate: PropTypes.func.isRequired,
     displayAlert: PropTypes.func.isRequired,
-    slugify: PropTypes.func.isRequired,
     // eslint-disable-next-line react/require-default-props
     mode: PropTypes.oneOf(['staff', 'articles', 'tags']).isRequired,
+    onChange: PropTypes.func,
```

Comment:
@frederikbrinck I made this optional as I for example don't need it in the create article one as I won't have a "there are changes" checker as of right now, just want your go ahead on it as it's your code